test/sql-tap/misc3.test.lua

```diff
@@ -510,7 +510,7 @@ test:do_execsql_test(
 test:do_execsql_test(
```

Comment:
The test tested CAST from bool to INTEGER, so now the test have to be removed

Reply:
Fixed